### PR TITLE
ADD: Active Validator Check on Import

### PR DESCRIPTION
**Pull request by mabasian**
This commit adds a new feature to our project, a popup user interface (UI) with functions for risk accept and cancel items. The popup UI is created to enhance the user experience by allowing users to perform actions quickly and efficiently without leaving the current page.

The new feature has been thoroughly tested and debugged to ensure its functionality. With this commit, the feature is ready to be wired to the backend.
part of: https://github.com/stereum-dev/ethereum-node/issues/628
part of: https://github.com/stereum-dev/ethereum-node/issues/630

### CODE REVIEW

File: launcher/src/backend/ValidatorAccountManager.js [hunk 1]
```diff
@@ -25,17 +28,54 @@ export class ValidatorAccountManager {
...
+      try {
+        for (const pubkey of pubkeys) {
+          let latestEpochsResponse = await axios.get(networkURLs[client.network] + pubkey + "/attestations");
+          if (
+            latestEpochsResponse.status === 200 &&
+            latestEpochsResponse.data.data.length > 0 &&
+            latestEpochsResponse.data.status !== /ERROR:*/
+          ) {
+            for (let i = 0; i < 2; i++) {
+              if (latestEpochsResponse.data.data[i].status === 0 && isActiveRunning.indexOf(pubkey) === -1) {
```
Comment (NeoPlays) [inline launcher/src/backend/ValidatorAccountManager.js:67]: guys you have to test your stuff properly 😅
`status` `1` means attested...
I was able to import an active validator on another machine without a problem.

Comment (gbayasgalan) [reply launcher/src/backend/ValidatorAccountManager.js:67]: true, that was for testing purpose, :)

File: launcher/src/components/UI/the-staking/DisplayValidators.vue [hunk 2]
```diff
@@ -752,6 +802,14 @@ export default {
     hideBDialog() {
       this.bDialogVisible = false;
     },
+    hideWDialog() {
+      this.riskWarning = false;
```
Comment (NeoPlays) [inline launcher/src/components/UI/the-staking/DisplayValidators.vue:806]: if you hide the dialog you also need to reset the button.
after i click cancel there is no button to import a key anymore

Comment (gbayasgalan) [reply launcher/src/components/UI/the-staking/DisplayValidators.vue:806]: @mabasian

File: launcher/src/backend/ValidatorAccountManager.js [hunk 3]
```diff
@@ -25,17 +28,54 @@ export class ValidatorAccountManager {
...
+  async checkActiveValidators(files, password, serviceID, slashingDB) {
+    this.batches = [];
+    this.batches = this.createBatch(files, password, slashingDB);
+    let services = await this.serviceManager.readServiceConfigurations();
+    let client = services.find((service) => service.id === serviceID);
+
+    let pubkeys = this.batches.map((b) => b.keystores.map((c) => JSON.parse(c).pubkey)).flat();
+    let isActiveRunning = [];
+
+    if (pubkeys.length < 11) {
```
Comment (NeoPlays) [inline launcher/src/backend/ValidatorAccountManager.js:52]: and we need to do something about this. infrom the user about this or something

File: launcher/src/backend/ValidatorAccountManager.js (hunk 1 above)
Comment (gbayasgalan) [reply launcher/src/backend/ValidatorAccountManager.js:67]: tnx

**Review by gbayasgalan — APPROVED** [review]

**Pull request merged**